src/pages/Locations/LocationsPage/hooks/useLocationsPage.ts

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
+import type { IonSearchbarCustomEvent } from '@ionic/core';
+import type { SearchbarChangeEventDetail } from '@ionic/react';
+import type { Location } from 'src/types';
+
+const useLocationsPage = (initialLocations: Location[]) => {
```

Comment:
this hook should be used at the LocationPageContainer level
it should not receive anything in arguments